Developer tools/Linter setup.md

```diff
@@ -6,7 +6,7 @@ In our projects we include several static analysis linters.
 
 #### PHP_CodeSniffer
 
-[PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer/) is a script that will tokenize your PHP files and based on selected rulesets, detect violations of coding standards.
+[PHP_CodeSniffer](https://github.com/squizlabs/PHP_CodeSniffer/) is a script that will tokenize your PHP files and based on selected rulesets, detect violations of set coding standards.

```

Comment:
@iruzevic can you just update this link (and if it's showing in other places in the handbook as well) to this:

https://github.com/PHPCSStandards/PHP_CodeSniffer

As the original repo has been abandoned (the development and the repo name on packagist are still the same so no worries about that).